python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -402,7 +402,12 @@ def __len__(self):
     # item.isA(SimpleItem)
     def isA(self, _what):
 
-        from Ganga.GPIDev.Base.Proxy import stripProxy
+        try:
+            from Ganga.GPIDev.Base.Proxy import stripProxy
+        except ImportError:
+            def dummyfunc(arg):
+                return arg
+            stripProxy = dummyfunc
```

Comment:
If this is to protect against putting proxied objects as default values in schemas then I'm not sure it's needed indeed. The only way it can happen is if one of us explicitly puts one in there in the code. The place we will have to be careful is to not put proxy objects that exist in physical Python code in there like `JobRegistrySliceProxy` or similar. There's already a check in `Schema._getDefaultValueInternal` (added by you I believe) to protect against it. If we end up with proxy objects in `_data` then we *want* it fail so that we can fix it.